node/generate-stackbrew-library.sh

```diff
@@ -61,8 +61,7 @@ aliases=(
 	[6.2.2]='6 latest'
 	[5.11.1]='5'
 	[4.4.5]='4'
-	[0.12.14]='0.12 0'
-	[0.10.45]='0.10'
+	[0.12.14]='0'
```

Comment:
I'm not 100% clear on what this is used for, but is it necessary to specifically alias 0.12.14 to '0'? As 0.12.15 is the most recent version, would we specify that anywhere?